automat/_methodical.py

```diff
@@ -68,14 +70,16 @@ def _transitionerFromInstance(oself, symbol, automaton):
     return transitioner
 
 
-@with_repr(['method'])
-@attributes(['automaton', 'method', 'symbol',
-             Attribute('collectors', default_factory=dict)],
-            apply_with_cmp=False)
+@attr.s(cmp=False, hash=False)
```

Comment:
`characteristic`'s `with_cmp` implied `__hash__`, but `attrs` decouples them.  I included `cmp=False` to keep the API the same.